Makefile

```diff
@@ -10,4 +10,11 @@ linux:
 	GOOS=linux GOARCH=amd64 go build -ldflags="$(FLAGS)" -o bin/ecs-upload-task-linux-$(VERSION) .
 
 windows:
-	GOOS=windows GOARCH=amd64 go build -ldflags="$(FLAGS)" -o bin/ecs-upload-task-win-$(VERSION) .
\ No newline at end of file
+	GOOS=windows GOARCH=amd64 go build -ldflags="$(FLAGS)" -o bin/ecs-upload-task-win-$(VERSION) .
+
+release-multi-arch:
+	mkdir -p bin
+	gox -os="linux darwin windows" -arch="amd64 arm64" -output="./bin/{{.Dir}}_{{.OS}}_{{.Arch}}"      
```

Comment:
Is this executing the compilation steps above or replacing them? I wonder if we need to keep them if we're now using `gox`

Reply:
Its replacing them. Mainly left them in as a legacy thing but probably don't need them anymore

Reply:
Yeah I think we should delete them if they're not used now or it'll be confusing

Reply:
So delete all the build steps or just the `all` step?